collections/_projects/mesh-channel-coordination.md

```diff
@@ -0,0 +1,128 @@
+---
+name: "Mesh channel coordination"
+desc: "Automatically select channels in mesh networks"
+requirements:
+ - "C programming skills"
+ - "Experienced Linux knowledge (usage and architecture)"
+ - "WLAN technology"
+ - "Game theory"
+ - "Graph theory"
+difficulty: "high"
+
+mentors:
+ - "Vincent Wiemann (CodeFetch)"
```

Comment:
please use your github user name here, so you can be linked on the projects page

Reply:
and if you have someone else who could mentor this, you can add them, too